hbase-common/src/test/java/org/apache/hadoop/hbase/util/RedundantKVGenerator.java

```diff
@@ -140,46 +139,45 @@ public RedundantKVGenerator(Random randomizer,
   private Random randomizer;
 
   // row settings
-  private byte[] commonPrefix;//global prefix before rowPrefixes
+  private byte[] commonPrefix; //global prefix before rowPrefixes
   private int numberOfRowPrefixes;
-  private int averagePrefixLength = 6;
-  private int prefixLengthVariance = 3;
-  private int averageSuffixLength = 3;
-  private int suffixLengthVariance = 3;
-  private int numberOfRows = 500;
+  private int averagePrefixLength;
+  private int prefixLengthVariance;
+  private int averageSuffixLength;
+  private int suffixLengthVariance;
+  private int numberOfRows;
 
-  //family
+  // family
   private byte[] family;
 
   // qualifier
-  private float chanceForSameQualifier = 0.5f;
-  private float chanceForSimilarQualifier = 0.4f;
-  private int averageQualifierLength = 9;
-  private int qualifierLengthVariance = 3;
+  private float chanceForSameQualifier;
+  private float chanceForSimilarQualifier;
+  private int averageQualifierLength;
+  private int qualifierLengthVariance;
 
-  private int columnFamilyLength = 9;
-  private int valueLength = 8;
-  private float chanceForZeroValue = 0.5f;
+  private int columnFamilyLength;
+  private int valueLength;
+  private float chanceForZeroValue;
 
-  private int baseTimestampDivide = 1000000;
-  private int timestampDiffSize = 100000000;
+  private int baseTimestampDivide;
+  private int timestampDiffSize;
```

Comment:
The values are overwritten later?

Reply:
They are both overwritten in the [constructor above](https://github.com/apache/hbase/pull/1030/files#diff-23663981b4ae838887df2503fa0586ccR113).

Reply:
Ok. Good.